.github/workflows/pytest.yml

```diff
@@ -89,35 +88,19 @@ jobs:
         with:
           fetch-depth: 2
 
-      - name: Install conda-lock environment with micromamba
-        uses: mamba-org/setup-micromamba@v2
+      - name: Install pudl environment using pixi
+        uses: prefix-dev/setup-pixi@v0.9.3
         with:
-          environment-file: environments/conda-lock.yml
-          environment-name: pudl-dev
-          cache-environment: true
-
-      - name: Log environment details
-        run: |
-          conda info
-          conda list
-          conda config --show-sources
-          conda config --show
-          printenv | sort
+          locked: true
```

Comment:
We install the locked environment everywhere in the workflows, otherwise they will automatically update the environment.